cosinnus_message/rocket_chat.py

```diff
@@ -728,25 +730,36 @@ def users_update(self, user, request=None, force_user_update=False, update_passw
         profile = user.cosinnus_profile
         rocket_email = user_data.get('emails', [{}])[0].get('address', None)
         # rocket_mail_verified = user_data.get('emails', [{}])[0].get('verified', None)
-        rocket_username = self._get_unique_username(profile)
-        if not rocket_username:
-            return
-        if force_user_update or user_data.get('name') != rocket_username or rocket_email != profile.rocket_user_email:
+        rocket_username = user_data.get('username')
+        username_changed = rocket_username is None or self.has_username_changed(profile, rocket_username)
```

Comment:
3. Can `user_data.get('username')` really be `None`, or is this check just cautionary? In what situation would it be `None`?

Reply:
I think this was just out of caution. It should never be  `None`. I'll removed this check.

Reply:
@2e2a no, you're welcome to leave the check in! rather do more checks and be future proof in case the API changes. maybe just add a comment along the lines of "x should always be present, but we check anyways"